frappe/core/doctype/file/file.py

```diff
@@ -498,7 +498,7 @@ def save_file(self, content=None, decode=False, ignore_existing_file_check=False
 		self.file_size = self.check_max_file_size()
 
 		if (
-			self.content_type and "image" in self.content_type
+			self.content_type and self.content_type is "image/jpeg"
```

Comment:
The `is` keyword is used to compare if the two variables are the same object, not the same value.

```suggestion
			self.content_type and self.content_type == "image/jpeg"
```

Reply:
Due to string interning, this would work, but I agree using that is not a good idea to rely on the pythoninternals 😁 

https://en.wikipedia.org/wiki/String_interning

Reply:
Oh, that's good to know. I would imagine this is generally true for comparing any immutable value to itself then.

Reply:
Thank you for the comments, I agree that we should explicit about this. I honestly didn't know there was a difference. I'll update my branch.